Public/Src/Cache/ContentStore/Distributed/NuCache/RocksDbContentLocationDatabase.cs

```diff
@@ -195,7 +195,12 @@ private BoolResult Load(OperationContext context, StoreSlot activeSlot, bool cle
                 Tracer.Info(context, $"Creating rocksdb store at '{storeLocation}'.");
 
                 var possibleStore = KeyValueStoreAccessor.Open(storeLocation,
-                    additionalColumns: new[] { nameof(Columns.ClusterState), nameof(Columns.Metadata) }, rotateLogs: true);
+                    additionalColumns: new[] { nameof(Columns.ClusterState), nameof(Columns.Metadata) },
+                    rotateLogs: true,

```

Comment:
BTW, an alternative fix for this is to actually change KeyValueStoreAccessor and always return a "Failure" with the right nested failure that caused the store to be inaccessible.

Reply:
Not necessarily, because we sometimes swallow the failures

---
In reply to: [322404029](https://github.com/microsoft/BuildXL/pull/849#discussion_r322404029) [](ancestors = 322404029)

Reply:
That was my point: to change KeyValueStoreAccessor to "remember" the failure. But this is not critical and the current fix works.

---
In reply to: [322404029](https://github.com/microsoft/BuildXL/pull/849#discussion_r322404029) [](ancestors = 322404029)